<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationWindow</class>
 <widget class="QMainWindow" name="AuthorizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация в системе</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Для продолжения работы с приложением&lt;br/&gt;вам необходимо авторизоваться&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип пользователя:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Преподаватель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Студент</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Логин:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userID_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="signInSystem_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Авторизироваться</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="connectionStatus_lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки соединения</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Информация о программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>userID_lineEdit</tabstop>
  <tabstop>userPassword_lineEdit</tabstop>
  <tabstop>signInSystem_pushButton</tabstop>
  <tabstop>connectionStatus_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
